Framework/PythonInterface/mantid/_plugins/__init__.py

```diff
@@ -0,0 +1,21 @@
+"""
+Plugins
+=======
+
+Defines Python objects that wrap the C++ DataObjects namespace. These are exported
```

Comment:
This isn't quite the right documentation for this subpackage as it's a little different to the others: Can we have something like:

"Defines Python objects that wrap classes from the framework defined as plugins. The classes defined here simply export the class types to Python but not any additional functionality. This allows Boost Python to automatically downcast a type to the actual leaf type, making it easier to work with these objects in Python."